constants.ts

```diff
@@ -0,0 +1,70 @@
+export const messages = {
+    cpp: {
+      language: "cpp",
+      message: "You think you are better than everyone just because your language have pointers?",
+    },
+    java: {
+      language: "java",
+      message: "Just use C++ like everyone else.",
+    },
+    javascript: {
+      language: "javascript",
+      message: "Don't you have to download a library to do that?",
+    },
+    rust: {
+      language: "rust",
+      message: "Why would you use that for leecode?",
```

Comment:
There's a typo here. Should've been `leetcode`

Reply:
> There's a typo here. Should've been `leetcode`

Thanks, just corrected it.